net/tailscale/files/tailscale.conf

```diff
@@ -1,7 +1,43 @@
-config settings 'settings'
-	option log_stderr '1'
-	option log_stdout '1'
+config tailscale 'settings'
+	# Set whether Tailcale is enabled or not
+	option enabled '0'
+	# Set the port to listen on for incoming Tailscale packets (default 41641)
 	option port '41641'
-	option state_file '/etc/tailscale/tailscaled.state'
-	# default to using nftables - change below to 'iptables' if still using iptables
-	option fw_mode 'nftables'
\ No newline at end of file
+	# Path to config file
+	option config_path '/etc/tailscale'
+	# Default to using nftables - change below to 'iptables' if still using iptables
+	option fw_mode 'nftables'
+	# Enable stdout log
+	option log_stdout '1'
+	# Enable stderr log
+	option log_stderr '1'
+	# Accept subnet routes that other nodes advertise
+	#option accept_routes '1'
+	# Provide a hostname to use for the device instead of the one provided by the OS
+	#option hostname ''
+	# Accept DNS configuration from the admin console. (default accept)
+	option accept_dns '1'
+	# Offer to be an exit node for outbound internet traffic from the Tailscale network
+	#option advertise_exit_node '1'
+	# Provide a Tailscale IP or machine name to use as an exit node
+	#option exit_node ''
+	# Expose physical subnet routes to your entire Tailscale network (e.g. 10.0.0.0/24)
+	#list advertise_routes ''
+	# Set to false to disable subnet route masquerading
+	#option site_to_site '1'
```

Comment:
How is masquerading and site to site related? This goes back to having your own config names instead of the upstream ones. E.g. I use Wireguard for site to site without any masquerading.

Reply:
I don't think UI and backend need to match 1-to-1: UI can be a user friendlier version of the configs, while configs match the upstream more closely. That is UI can call this feature whatever (e.g. Site to site), and wrap multiple features under a single checkbox (the ones from the link above). The backend (i.e. this package) can be used without the UI, so having UI-specific configs doesn't make sense.